### ISSUE TITLE
On your dataset

### ISSUE THREAD

**Issue by yjjuan** [AutonomousFieldRoboticsLab/gopro_ros#3]
Hi joshi-bharat,
thanks for your sharing of this wonderful work. I tried to download the dataset you shared at [your google drive](https://afrl.cse.sc.edu/afrl/resources/datasets/), but only found the dataset of ship-wreck. Is there any possibility I could download the dataset of cave and cavern to reproduce the result in your paper?

**Comment by joshi-bharat** [comment]
Yes, we were supposed to put them on the website. Until we finish adding them to the website, I will add a link to the drive folder this week.  Sorry about the delay. 

Thanks.

**Comment by yjjuan** [comment]
Hi,
I just come across your drive folder, and still only found ship-wreck dataset. How to obtain other datasets? 
Thanks.

**Comment by joshi-bharat** [comment]
Yeah, we are in the process of releasing the dataset. I will notify you of this. Thanks

**Comment by onurbagoren** [comment]
Hello -- is there any update on releasing the dataset? The link on the first comment unfortunately doesn't work to get the videos and calibration data

**Comment by AravindhPadmanabhan** [comment]
Hello @joshi-bharat, thanks for your interesting work! Any updates on your dataset? I am not able to find the sequences anywhere.
